src/azure-cli/azure/cli/command_modules/feedback/tests/latest/test_feedback.py

```diff
@@ -190,14 +192,20 @@ def _run_cmd(self, command, checks=None, expect_failure=False):
         cli_ctx = get_dummy_cli()
         cli_ctx.logging.command_log_dir = self.temp_command_log_dir
 
-        result = execute(cli_ctx, command, expect_failure=expect_failure).assert_with_checks(checks)
+        # hotfix here for untouch feedback's code to avoid introducing possible break change.
+        # unregister the event for auto closing CLI's file logging after execute() is done
+        cli_ctx.unregister_event(EVENT_CLI_POST_EXECUTE, cli_ctx.logging.deinit_cmd_metadata_logging)
```

Comment:
Have to unregister EVENT_CLI_POST_EXECUTE event to make cli instance not to clean up file logger handler. Such that to make this `logger.error("There was an error during execution.")`  log to log file.

This is a hotfix. Otherwise, have to modify the code of feedback command.